Algebroids/makedoc.g

```diff
@@ -21,12 +21,16 @@ AutoDoc( rec(
         LaTeXOptions := rec(
             LateExtraPreamble := """
                 \usepackage{mathtools}
+                \usepackage{tikz-cd}
                 \DeclareUnicodeCharacter{2205}{\ensuremath{\emptyset}}
                 \DeclareUnicodeCharacter{2C76}{\ensuremath{\vdash}\!\!}
                 \DeclareUnicodeCharacter{2192}{\ensuremath{\!\!\rightarrow\!}}
                 \DeclareUnicodeCharacter{21AA}{\ensuremath{\!\!\hookrightarrow\!}}
                 \DeclareUnicodeCharacter{21A0}{\ensuremath{\!\!\twoheadrightarrow\!}}
                 \DeclareUnicodeCharacter{2B47}{\ensuremath{\!\!\xrightarrow{\sim}\!}}
+                \DeclareUnicodeCharacter{227B}{\ensuremath{\succ}}
+                \DeclareUnicodeCharacter{2022}{\ensuremath{\bullet}}
+                \DeclareUnicodeCharacter{2297}{\ensuremath{\otimes}}
```

Comment:
This should be produced by PackageJanitor.

Reply:
I will submit a PR to `PackageJanitor`  including the extra latex-declarations after merging this PR.

Reply:
Thanks, it is only important that these lines are created by `PackageJanitor` :)